docs/source/user_guide/groups.rst

```diff
@@ -179,9 +180,9 @@ expanded as you add samples to a grouped dataset.
 
 .. note::
 
-    Grouped datasets may contain a mix of images, videos, and point clouds, but
-    FiftyOne strictly enforces that each **slice** of a grouped dataset must
-    have a homogeneous media type.
+    Grouped datasets may contain a mix of different modalities (e.g., images,
+    videos, and 3D scenes), but FiftyOne strictly enforces that each **slice**
+    of a grouped dataset must have a homogeneous media type.
```

Comment:
Should we mention that there cannot be more than one 3d slice?

Reply:
Hmm, feels a bit too specific for this context but not sure where else to put it